### PR TITLE
chor(release): Release changelog for 3.9.0

### PR DESCRIPTION
**Pull request by GMishx**
Change log updates for release 3.9.0

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+fossology (3.9.0-1) unstable; urgency=low
+
+  * New upstream release
+  * https://github.com/fossology/fossology/releases/tag/3.9.0 for details
+
+ -- Gaurav Mishra <mishra.gaurav@siemens.com>  Mon, 30 Nov 2020 11:11:11 +0530
```
Comment (mcjaeger) [inline debian/changelog:6]: interesting date ... r u planning to wait or is just a guess?

**Review by mcjaeger — APPROVED** [review]
contents looks good, also tested the instance and check if the version string and basic clearing works. did *not* test debian package building. waiting for finishing travis now.

File: debian/changelog (hunk 1 above)
Comment (GMishx) [reply debian/changelog:6]: I just left the date so we can do tests on master before tagging. If no issues were found, I will change the date before tagging.

**Comment by GMishx** [comment]
Found a small bug in spasht.

```
PHP Fatal error:  Uncaught Error: Cannot use object of type Fossology\\Lib\\Data\\Spasht\\Coordinate as array in /usr/local/share/fossology/spasht/ui/ui-spasht.php:402
Stack trace:
#0 /usr/local/share/fossology/spasht/ui/ui-spasht.php(210): ui_spasht->checkAdvanceSearch(Object(Fossology\\Lib\\Data\\Spasht\\Coordinate), '3.4.1', '', '', '')
#1 /usr/local/share/fossology/lib/php/Plugin/FO_Plugin.php(383): ui_spasht->Output()
#2 /usr/local/share/fossology/lib/php/Plugin/FO_Plugin.php(467): FO_Plugin->getResponse()
#3 /usr/local/share/fossology/www/ui/index.php(64): FO_Plugin->execute()
#4 {main}
  thrown in /usr/local/share/fossology/spasht/ui/ui-spasht.php on line 402, referer: http://localhost:8080/repo/?mod=spashtbrowser&upload=5&item=90
```

**Pull request merged**